.circleci/config.yml

```diff
@@ -1,241 +1,216 @@
 version: 2.1
 
-slack/notify-on-failure:
-  only_for_branches: "master"
-
+# -------------------------
+#   ORBS
+# -------------------------
 # https://circleci.com/orbs/registry/
 orbs:
   browser-tools: circleci/browser-tools@1.0.1
-  slack: circleci/slack@3.4.2
 
 # -------------------------
-#   PARAMETERS
+#   PIPELINE PARAMETERS
 # -------------------------
 parameters:
-  circleci_service_account:
+  circle-service-account:
     type: string
     default: "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com"
-  workbench_image:
+  workbench-image:
     type: string
     default: "allofustest/workbench:buildimage-0.0.19"
-  mysql_image:
+  db-image:
     type: string
     default: "circleci/mysql:5.7"
 
 # -------------------------
 #   ANCHORS
 # -------------------------
 anchors:
-  java_env: &java_env
-    # As best I can tell (dmohs, 7 Feb '17), this is the only way to set a memory limit that Java
-    # processes executed within CircleCI's docker containers will respect. Very helpful resource:
-    # https://circleci.com/blog/how-to-handle-java-oom-errors/
-    #
-    # In Feb 2017, this was set to 3G. But in Feb 2019 (RW-2194) we started seeing OOM errors,
-    # so we bumped this down further to 2G.
+  default-environment: &default-environment
     JAVA_TOOL_OPTIONS: -Xmx2g
     # See: https://support.circleci.com/hc/en-us/articles/360021812453-Common-Android-memory-issues
     GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2 -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy=in-process
     TERM: dumb
-    MYSQL_ROOT_PASSWORD: ubuntu
 
-  mysql_env: &mysql_env
+  db-environment: &db-environment
     MYSQL_ROOT_PASSWORD: ubuntu
     MYSQL_USER: ubuntu
     MYSQL_PASSWORD: ubuntu
 
-  e2e_env: &e2e_env
+  puppeteer-test-environment: &puppeteer-test-environment
```

Comment:
nit: puppeteer-environment seems sufficient to me, the "-test-" infix is sort of implied.